presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/BenchmarkRunnerConfig.java

```diff
@@ -24,13 +24,15 @@
 import java.util.Set;
 
 import static com.facebook.presto.benchmark.source.DbBenchmarkSuiteSupplier.BENCHMARK_SUITE_SUPPLIER;
+import static java.lang.Boolean.FALSE;
 
 public class BenchmarkRunnerConfig
 {
     private String testId;
     private String benchmarkSuiteSupplier = BENCHMARK_SUITE_SUPPLIER;
     private Set<String> eventClients = ImmutableSet.of("json");
     private Optional<String> jsonEventLogFile = Optional.empty();
+    private boolean proceedWithFailures = FALSE;
```

Comment:
You should not need to assign a `java.lang.Boolean.FALSE` to a `boolean`- you should just use `false`.
Since `false` is the default value for `boolean`, you don't even need the initialization .
```Java
 private boolean proceedWithFailures
```